conf/deployment/fusion_aas/managed_3az_provider_qe_3m_3w_m52x.yaml

```diff
@@ -3,6 +3,7 @@ ENV_DATA:
   platform: 'fusion_aas'
   deployment_type: 'managed'
   cluster_type: 'provider'
+  cluster_namespace: 'fusion-storage'
   multi_availability_zones: True
```

Comment:
If I understand correctly, namespace change to the config value isn't restricted to managed services but will happen to the whole ODF. Shouldn't we then include this value in the config files for other types of deployment?

Reply:
It is already part of a default config and on some places used for other platforms too. https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml#L135